csharp/rules.bzl

```diff
@@ -34,3 +36,28 @@ def swig_native_csharp_library(name, nullable_context, target_frameworks, target
         actual = "__" + name,
         visibility = visibility,
     )
+
+
+def swig_native_nuget_pack(name, id, libs, native_libs, target_framework, nuspec_template, platforms, visibility, files={}, **kwargs):
+    # generate identical libraries with different maven coordinate tags, since we can't 'select' tags
+    for platform in platforms.values():
+        nuget_pack(
```

Comment:
Does this rule support `target_compatible_with`? See `//c/assemble_targz` for example

Reply:
`target_compatible_with` returns a error if I even try to build a target for another platform. As I want to build several targets following the java way and then choose only one, it doesn't suite me. 

But I could just call for `nuget_pack` 5 times in the `BUILD` file and then make an alias over it like the c/cpp drivers do, if you want me to. Maybe it's a hint to try refactoring java as well. What do you think?